data/json/overmap/overmap_terrain/overmap_terrain_transportation.json

```
@@ -290,11 +290,29 @@
     "color": "dark_gray",
     "see_cost": 5,
     "extras": "subway",
+    "mondensity": 2,
+    "mapgen_straight": [ { "method": "builtin", "name": "subway_straight" } ],
+    "mapgen_curved": [ { "method": "builtin", "name": "subway_curved" } ],
+    "mapgen_end": [ { "method": "builtin", "name": "subway_end" } ],
+    "mapgen_tee": [ { "method": "builtin", "name": "subway_tee" } ],
+    "mapgen_four_way": [ { "method": "builtin", "name": "subway_four_way" } ],
+    "spawns": { "group": "GROUP_SUBWAY", "population": [ 2, 3 ], "chance": 50 },
```

```suggestion
    "spawns": { "group": "GROUP_SUBWAY", "population": [ 1, 3 ], "chance": 63 },
```
IIRC, some of the ones in that group may wind up fighting each other, so to increase the frequency increasing the chance instead may be preferable.